service-automation/src/com/appyschool/common/CommonUtils.java

```diff
@@ -37,7 +37,8 @@
 	 * @param newHeader
 	 * @return
 	 */
-	public static RequestSpecification getRequestSpec() {
```

Comment:
I see no changes in this file, please revert this.

Don't check-in a file until there is a change in it.

Reply:
Okay.